contracts/contracts/token/OUSD.sol

```diff
@@ -445,50 +418,61 @@ contract OUSD is Initializable, InitializableERC20Detailed, Governable {
         view
         returns (uint256)
     {
-        if (nonRebasingCreditsPerToken[_account] != 0) {
-            return nonRebasingCreditsPerToken[_account];
+        if (alternativeCreditsPerToken[_account] != 0) {
+            return alternativeCreditsPerToken[_account];
         } else {
             return _rebasingCreditsPerToken;
         }
     }
 
     /**
-     * @dev Is an account using rebasing accounting or non-rebasing accounting?
-     *      Also, ensure contracts are non-rebasing if they have not opted in.
+     * @dev Before a `rebaseOptIn` or non yield delegating token `transfer` can be executed contract
+     *      accounts need to have a more explicitly defined rebasing state set.
+     *
+     *      Contract account can be in the following states before `autoMigrate` is called:
+     *      1. Under any token contract codebase they haven't been part of any token transfers yet
+     *         having rebaseState `NotSet` and `alternativeCreditsPerToken == 0`
+     *      2. Under older token contract codebase they have the default rebaseState set to `NotSet` and
+     *         the codebase has "auto-migrated" them by setting the `alternativeCreditsPerToken` to some
+     *         value greater than 0.
+     *      3. Contract has under any token contract codebase explicitly requested to be opted out of rebasing
+     *
+     *     Case 1. Needs to be migrated using autoMigrate to a nonRebasing account.
+     *
+     *     Note: Even with this _autoMigrate function in place there will still be Case 2 accounts existing that
+     *           will behave exactly like RebaseState StdNonRebasing account, and still having their rebase state
+     *           set to `NotSet`
+     *
      * @param _account Address of the account.
      */
-    function _isNonRebasingAccount(address _account) internal returns (bool) {
-        bool isContract = Address.isContract(_account);
-        if (isContract && rebaseState[_account] == RebaseOptions.NotSet) {
-            _ensureRebasingMigration(_account);
+    function _autoMigrate(address _account) internal returns (bool) {
```

Comment:
It looks like the return value is defined but never actually used

Reply:
fixed: https://github.com/OriginProtocol/origin-dollar/pull/2298/commits/bbb3f45defdcb7b52460fb06badb07620b4ecdd8